content/rs/databases/active-active/app-failover-active-active.md

```diff
@@ -1,13 +1,18 @@
 ---
-Title: Active-Active region application failover
+Title: Application failover with Active-Active databases
 linkTitle: App failover
-description:
-weight: 1
+description: How to failover your application to connect to a remote replica.
+weight: 60
 alwaysopen: false
 categories: ["RS"]
-aliases: /rs/developing/crdbs/region-application-failover-active-active/
+aliases: [
+    /rs/developing/crdbs/region-application-failover-active-active/,
+    /rs/databases/active-active/app-failover-active-active.md,
+    /rs/databases/active-active/app-failover-active-active/,
+]
+
```

Comment:
You can change the headers to sentence case and maybe change "Detecting Failure" to "Detect failure".